addon/components/ember-collection.js

```diff
@@ -50,20 +38,25 @@ export default Ember.Component.extend({
     Ember.set(this, 'cells', Ember.A([]));
     this.cellMap = Object.create(null);
   },
+
   didInitAttrs() {
-    this.buffer = this._maybeMutAttr('buffer', 5);
-    this.offsetX = this._maybeMutAttr('offset-x', 0);
-    this.offsetY = this._maybeMutAttr('offset-y', 0);
-    this.width = this._maybeMutAttr('width', 0);
-    this.height = this._maybeMutAttr('height', 0);
+    this.buffer = this.getAttr('buffer') | 5;
```

Comment:
this line doesn't make sense, why bitwise-or with 5? the idiom with `| 0` means int value coercion not default to 0.